site/source/locales/ui-fr.yaml

```
@@ -1353,7 +1353,7 @@ pages:
       title: Simulateur de revenus pour professions libérales Cipav
     comparaison:
       meta:
-        description: Auto-entrepreneur, EI/EURL ou SASU ? Simulez les différences de
+        description: Auto-entrepreneur, EI/EURL ou SAS(U) ? Simulez les différences de
```

C'est le seul endroit ou je remettre SASU, car le cas SAS avec un seul associé qui se verse une rémunération est exceptionnel par rapport au reste.
```suggestion
        description: Auto-entrepreneur, EI/EURL ou SASU ? Simulez les différences de
```

---

Modifié !